test/fixtures/track-basic.json

```diff
@@ -32,7 +32,7 @@
       "$browser": "Safari",
       "$os": "iPhone OS",
       "$os_version": "8.1.3",
-      "$browser_version": "7.1.0.7",
+      "$browser_version": 7.1,
```

Comment:
Does it matter that a browser version could be `1.0.0`? or do we lop off that precision?